node/src/proxy_server/mod.rs

```diff
@@ -532,7 +532,7 @@ impl ProxyServer {
                                 Err(e) => {
                                     error!(
                                         logger,
-                                        "Neighborhood refused to answer route request: {}", e
+                                        "Neighborhood refused to answer route request: {:?}", e
```

Comment:
Please check if it really needs to be changed to be able to accept Debug instead of Display. Try to google the difference of them for Rust. :) 

Again if it can stay the former way make it look back like that. Otherwise we'll accept.

Reply:
![image](https://user-images.githubusercontent.com/89403560/131414103-1c0b2605-ee3d-4b8d-a866-b601d55eea52.png)
Mailbox error doesn't work with {}, but it does display with {:?}

Reply:
You are too pro-active. Here it reveal on you that you abide by the hints by the Rust plug-in. The red line in the code seemingly highlight a real error but not always it's true. You should not listen to this blindly. 

Always follow the hints of the compiler, he's a smarter guy. 

You're told here to add {:?} because they say the trait Display isn't implemented. Well, if you go deep into the structure you'll find that MailboxError is equipped with a derive trait. That's the trait which is easiest to impl, you just name it after the keyword "derive".

Next, if you click on the trait in the derive impl, you'll see that the trait has supertraits - alias traits whose methods it inherits too. You can see Display among those supertraits. Here we are, the Display trait is actually in the place. 

Please remove the {:?} formatting mark, there is no need for it. We could call it a bug of the Rust plugin.